hub/cli/auth.py

```diff
@@ -5,6 +5,7 @@
 """
 
 import click
+import hub
```

Comment:
Why do we need to import hub here though?

Reply:
solved.